frontend/src/components/management-data/DocumentUnitManagementData.vue

```diff
@@ -1,14 +1,37 @@
 <script lang="ts" setup>
 import { storeToRefs } from "pinia"
+import { ref, watch } from "vue"
 import DocumentUnitDeleteButton from "@/components/DocumentUnitDeleteButton.vue"
+import DocumentUnitHistoryLog from "@/components/management-data/DocumentUnitHistoryLog.vue"
 import DuplicateRelationListItem from "@/components/management-data/DuplicateRelationListItem.vue"
 import ManagementDataMetadata from "@/components/management-data/ManagementDataMetadata.vue"
 import TitleElement from "@/components/TitleElement.vue"
 import DocumentUnit from "@/domain/documentUnit"
+import DocumentUnitHistoryLogService from "@/services/documentUnitHistoryLogService"
 import { useDocumentUnitStore } from "@/stores/documentUnitStore"
 import IconCheck from "~icons/ic/baseline-check"
 
 const { documentUnit } = storeToRefs(useDocumentUnitStore())
+const data = ref()
+const error = ref()
+const isLoading = ref()
```

Comment:
Die haben vom Typ her `Ref<any>`
```suggestion
const historyLogs = ref<DocumentationUnitHistoryLog[]>()
const error = ref<ResponseError>()
const isLoading = ref(false)
```
Und ich würde statt `data` einfach direkt sagen, was drin ist